lms/djangoapps/instructor/tests/test_legacy_raw_download_csv.py

```diff
@@ -67,14 +67,17 @@ def test_download_raw_grades_dump(self):
 '''
         self.assertEqual(body, expected_csv, msg)
 
-    def test_grade_summary_data(self):
+    def get_expected_data(
```

Comment:
Consider changing this to `get_expected_grade_data()`, though since it's local to this class it may be unnecessary.